Directory.Build.targets

```diff
@@ -1,2 +1,87 @@
 <Project>
+
+  <ItemGroup Label="Package Versions">
```

Comment:
Please revert this file. In one of the latest commits I removed all shared versions - #951 .

Reply:
Commit: d4ae8d388b2d23f2395ca46391fffba537531dfb [d4ae8d3]
Parents: 1828caa3c9
revert build targets, edit csproj